### PR TITLE
Emib next button

### PR DESCRIPTION
**Pull request by MichaelCherry**
Added check to determine the current page and look up the relevant text for the next button accordingly.
New French text has been prefixed with "FR"

### CODE REVIEW

File: frontend/src/components/eMIB/Emib.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ let STRINGS = new LocalizedStrings({
     howToPageTitle: "HowTo Page",
     emailInstructions: "Email Instuctions",
     taskInstructions: "Task Instuctions",
+    howToNextButton: "Start timer and test",
```
Comment (fnormand01) [inline frontend/src/components/eMIB/Emib.jsx]: I find the length of this button a little bit long, especially in french. Maybe we could just have a button 'Start Test' and add a note saying that there will be a timer during the test?

**Comment by fnormand01** [comment]
I translated your two new buttons in French. See commit #https://github.com/code-for-canada/project-thundercat/pull/58/commits/a9dab933f778ea86c7038dfc83a9b955a7fefa51

Also, the changes look good to me. However, I added a comment related to the 'Start timer and test' button title length.

**Comment by fnormand01** [comment]
@MichaelCherry, I updated the button labels, as discussed offline. Is that fine with you?
See commit #https://github.com/code-for-canada/project-thundercat/pull/58/commits/cad82aefb2119efd327e9db6183f937e0d8071fb

File: frontend/src/components/eMIB/Emib.jsx (hunk 1 above)
Comment (MichaelCherry) [reply frontend/src/components/eMIB/Emib.jsx]: discussed offline; decided to crop button to just say "Start Test". Other PSC tests only say "Start Test"

**Comment by MichaelCherry** [comment]
Changes look good; discussed offline that most PSC tests say "Start test" rather than the much longer and more unwieldy "Start test and timer".

**Comment by fnormand01** [comment]
**Tests**
---
image

**Pull request merged**